core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -538,7 +543,7 @@ public void testNamespaceWithDot() {
   public void testBasicCreateTable() {
     C catalog = catalog();
 
-    TableIdentifier ident = TableIdentifier.of("ns", "table");
+    TableIdentifier ident = TableIdentifier.of("ns", "nstable");
```

Comment:
nit: I'd probably then call this `tbl` instead of `nstable` because the `ns` in the same could be a bit misleading. Same for all the other places further below